Erster Entwurf für englische Version der Spielregeln (#11)
Bierball-Club-Potsdam/docs: Rules.md

# Rules of the Game

## Preface

Bierball is a sport where the enjoyment of all players is the main focus. All participants should be treated with respect at all times. Fair play is expected from all participants.

## § 1 Objective of the Game

(1) The game is considered over as soon as one team has completely emptied their drinks.

> (2) A drink is considered emptied when the drinking container is held upside down over the player’s head without a significant amount of liquid escaping from the container. If liquid does escape, the team must hit the target again for the container to be considered emptied. If a larger amount of liquid escapes, additional penalties may be applied (see § 6).

Reviewer: ```suggestion
(2) A drink is considered emptied when the drinking container is held upside down over the player’s head without a significant amount of liquid leaving the container. If this is not the case, the team must hit the target again for the container to be considered emptied. If a larger amount of liquid escapes, additional penalties may be applied (see § 6).
```

## § 2 Teams

The game is played with two equally sized teams, each consisting of at least one person.

## § 3 Equipment

(1) The following equipment is required to play:

1. A throwing object
2. A target

> 3. A drinking container per player

Reviewer: Klingt alles so falsch im Englischen :D 
(Aber ich hab auch keine bessere Idee)

(2) The throwing object should be a nearly round item that can be easily thrown with one hand.

(3) The target must resemble a 1.0-liter plastic bottle in size and shape. It must be weighted in a way (e.g., by filling it with water) so that it does not fall over easily from minor forces like wind. However, it should be possible to knock over the target with the throwing object.

> (4) Drinks are consumed from 0.5-liter bottles made of glass or another non-elastic material. The opening must have a diameter of approximately 17 mm. This includes (but is not limited to) commercially available beer bottles with crown corks (NRW bottle, Longneck bottle) as well as bottles with swing tops or screw caps. The drinks must contain carbon dioxide. It is explicitly allowed to consume beverages that do not contain alcohol or are not classified as beer.

Reviewer: ```suggestion
(4) Drinks are consumed from 0.5-liter bottles made of glass or another non-elastic material. The opening must have a diameter of approximately 17 mm. This includes (but is not limited to) commercially available beer bottles with crown corks (e.g. German "NRW-Flasche", "Langhalsflasche") as well as bottles with swing tops or screw caps. The drinks must contain carbon dioxide. It is explicitly allowed to consume beverages that do not contain alcohol or are not classified as beer.
```

Author: Zumindest auf dem englischen Wikipedia werden auch die übersetzten Begriffe genutzt, Longneck ist wohl auch ein internationales Format: 
https://en.wikipedia.org/wiki/Beer_bottle?wprov=sfti1#Germany

Reviewer: Na dann will ich nichts gesagt haben. Top Recherchearbeit!

(5) Each player has exactly one drinking container and drinks only from this container.

## § 4 Playing Field

(1) The target is placed in the middle of the playing field.

(2) From there, 6 meters (~7 steps) are measured in opposite directions to determine the baselines (also called "throwing lines") for the two teams.

(3) The teams' baselines run parallel.

(4) The target is located in the center of the two baselines.

## § 5 Procedure

(1) There is always exactly one team trying to knock over the target with the throwing object ("throwing team") and one team responding ("setting team"). After each throw, the roles switch.

(2) Before the game begins, a coin toss or a game of rock-paper-scissors between two members of the playing teams decides the first throw. The winning team decides which team throws first.

> (3) Before each throw, it must be ensured that the opposing team is ready. This is done by raising a hand or calling out by the throwing team, which the setting team confirms by raising a hand or calling out in return. If this rule is ignored, the throw is annulled, and the throwing right goes to the opposing team.

Reviewer: ```suggestion
(3) Before each throw, it must be ensured that the opposing team is ready. This is done by raising a hand or call-out by the throwing team, which the setting team confirms by raising a hand or calling out in return. If this rule is ignored, the throw is annulled, and the throwing right goes to the opposing team.
```

(5) The rest of the throwing team is free to choose their posture but may not touch the drinking containers before the target has fallen.

(6) If the target falls before the throwing object has crossed the setting team's baseline, the throwing team may begin drinking. Drinking continues until the target is successfully reset and all members of the setting team, along with the throwing object, are behind their baseline. The completion of this process is signaled with a clearly audible "Stop" call. The throwing team must immediately stop all drinking actions and place the bottles back on the ground. The mouth may be pressed against the bottle opening to prevent foam from escaping (see §6 (1)). The opening must point upwards.

(7) Within a team, the throwing person changes after each throw.

## § 6 Penalties

(1) In the following cases, the opposing team is allowed to take an extra sip from their drinking container ("penalty sip"):
- A drink foams over.
- Liquid ends up somewhere other than the mouth of the drinking person.

> - A team has drunk, although the throw was invalid and annulled.

Reviewer: ```suggestion
- A team drank, although the throw was invalid and annulled.
```
